src/test/java/games/strategy/util/UtilTest.java

```diff
@@ -0,0 +1,28 @@
+package games.strategy.util;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class UtilTest {
+
```

Comment:
Empty string is a valid argument, it produces a hash like every other valid string.
null will probably lead to a NullPointerException, we are calling string.toBytes